docs/index.md

```diff
@@ -67,12 +66,12 @@ The easiest way to get started with Swarm is to use the
 [official Docker image](https://registry.hub.docker.com/_/swarm/).
 
 ```bash
-docker pull swarm
+$ docker pull swarm
```

Comment:
I feel like the style guide @jamtur01 references above is pretty straightforward and I don't see a compelling reason to deviate, unless I'm missing something. Don't the brackets `< >` make it clear it's a "template" command?